### DISCUSSION TITLE
Can the frontend be open sourced?

### DISCUSSION THREAD

**Discussion by TRSWNCA** [Equim-chan/mjai-reviewer#76] [Q&A]
I want to add some features like analyzing the play inside tenhou, just like by NAGA using CUSTOM URL here:
![image](https://user-images.githubusercontent.com/43987494/182758642-7c8f6437-ff67-4f35-8416-f0c75206c053.png)

目前线上预览的代码可以开源吗，我想增加在牌谱这直接解析的功能

**Comment by Equim-chan** [comment]
这个功能我有考虑过，不过目前还没有计划实现。

**Reply by TRSWNCA** [reply]
我可以为他写个前端/修改下业务层的代码适配
但是我在阿里云的机子实在太差，很难自己在机子上训练一个
如果能够开源的话挺好的

**Reply by Equim-chan** [reply]
这个前端指的具体是什么的呢？是说的在线服务的 portal，还是类似牌谱播放器一样、然后能在旁边展示当前局面的数据的 web UI？

**Comment by TRSWNCA** [comment]
前者，就是输入URL的那里。看局面和何切的界面应该是本项目的内容吧。



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年08月27日 23:39 |
| 收件人 | ***@***.***> |
| 抄送至 | Yichi ***@***.******@***.***> |
| 主题 | Re: [Equim-chan/mjai-reviewer] Can the frontend be open sourced? (Discussion #76) |

这个前端指的具体是什么的呢？是说的在线服务的 portal，还是类似牌谱播放器一样、然后能在旁边展示当前局面的数据的 web UI？

**Reply by Equim-chan** [reply]
那个完全是手写的，也只是一个简单的表单提交而已，然后后端根据填入的内容写好参数，启动相应子进程。写得比较粗糙所以也没有开源。

**Comment by TRSWNCA** [comment]
是的，那我现在我的机器上试试吧，如果合适我会提交的


| |
cyc
|
|
***@***.***
|
On 8/28/2022 ***@***.***> wrote：

那个完全是手写的，也只是一个简单的表单提交而已，然后后端根据填入的内容写好参数，启动相应子进程。写得比较粗糙所以也没有开源。
